mingw_smoketests.py

```diff
@@ -214,7 +214,10 @@ def test_platform_things(self):
         ext_suffixes = importlib.machinery.EXTENSION_SUFFIXES
         self.assertTrue(ext_suffix in ext_suffixes)
         self.assertTrue(".pyd" in ext_suffixes)
-        self.assertEqual(sys.winver, ".".join(map(str, sys.version_info[:2])))
+        if '64' in sys.version:
+            self.assertEqual(sys.winver, ".".join(map(str, sys.version_info[:2])))
+        else:
+            self.assertEqual(sys.winver, ".".join(map(str, sys.version_info[:2])) + '-32')
```

Comment:
this smoketest will fail on arm32 and arm64 now

Reply:
yeah will update

Reply:
updated

Reply:
Can you build and test this PR on Arm?

Reply:
I can definitely do so on arm64.  I can probably figure something out cross-compiling for arm32.